test/litmus/type/date_time_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule Litmus.Type.DateTimeTest do
+  use ExUnit.Case, async: true
+  doctest Litmus.Type.DateTime
+
+  alias Litmus.Type
+
+  describe "validate_field/3" do
+    test "converts a string to a datetime" do
+      field = "start_date"
+      data = %{"start_date" => "1990-05-01T06:32:00Z"}
+
+      {:ok, expected_date_time, _} = DateTime.from_iso8601(data["start_date"])
+      expected_data = %{"start_date" => expected_date_time}
+
+      type = %Type.DateTime{
+        required: true
+      }
+
+      assert Type.DateTime.validate_field(type, field, data) == {:ok, expected_data}
```

Comment:
could we replace `field` with `"start_date"` above and remove the `field` declaration above?

Reply:
Alternatively, since the same `field` is used throughout all these tests, maybe we just make it a module attribute?